uberfire-security/uberfire-security-management/uberfire-security-management-keycloak/src/main/java/org/uberfire/ext/security/management/keycloak/BaseKeyCloakManager.java

```diff
@@ -102,12 +102,20 @@ protected void loadConfig( final ConfigProperties config ) {
     }
 
     protected synchronized Keycloak getKeyCloakInstance() {
-        if (this.keycloak == null) {
+        if ( null == this.keycloak ) {
```

Comment:
will remove the syncronization here yes, does not makes sense. About the notation of the if... just notation, no matter for me to revert it :)